app/Http/Controllers/GameController.php

```diff
@@ -10,34 +11,55 @@
 use App\Http\Requests\UpdateGameStateRequest;
 use App\Models\Game;
 use Carbon\Carbon;
-use OutOfBoundsException;
+use Illuminate\Support\Collection;
 
 class GameController extends Controller
 {
-    public function getUsersInGame($gameId)
+    public function index()
     {
-        return Game::find($gameId)->users();
+        return view('game.index', ['games' => Game::all()]);
     }
 
-    public function getLootInGame($gameId)
+    public function get(Game $game)
     {
-        return Game::find($gameId)->loots;
+        return $game;
     }
 
-    public function getStatusInGame($gameId)
+    public function getUsers(Game $game)
```

Comment:
Bij het ophalen van `Users` wordt nu geen `role` mee gegeven. Deze is terug te vinden in de `InviteKey`. Dit is de response die ik vanuit de api krijg:

```
[{"id":3,"username":"Johan","location":"51.751290,5.500407","created_at":"2021-04-22T08:23:37.000000Z","updated_at":"2021-04-22T08:23:37.000000Z"},{"id":4,"username":"Rik","location":"51.769166,5.548282","created_at":"2021-04-22T08:23:37.000000Z","updated_at":"2021-04-22T08:23:37.000000Z"}]
```

Reply:
En de invite keys onthouden zo te zien ook hun user id. Dus eventueel kan er voor het ophalen van de role van een user gebruik worden gemaakt van de inviteKey() functie, waarvan dan de role wordt opgevraagd.